meta/documents/user_guide/de/initial_setup.md

```diff
@@ -33,7 +33,7 @@ Die Schritte Autorisierung sind wie folgt:
 
     ![linked clients](../../../images/de/plugin_sets_linked_clients.png)
 
-3. Klicken Sie auf die Schaltfläche `bearbeiten` ![pencil button](../../../images/common/button_pencil.png) für das gewünschte Plugin-Set.
+3. Klicken Sie auf das gewünschte Plugin-Set..

```

Comment:
```suggestion
3. Klicken Sie auf das gewünschte Plugin-Set.
```
I assume the double periods wasn't intentional and it doesn't mean something in German?